.circleci/config.yml

```diff
@@ -169,23 +173,21 @@ workflows:
           should_build: << pipeline.parameters.model >>
           requires:
             - install-yarn
+      - build-lint-test:
+          name: designer-build-lint-test
+          package_name: designer
+          should_build: << pipeline.parameters.designer >>
+          requires:
+            - model-build-lint-test
       - build-lint-test:
           name: engine-build-lint-test
           package_name: engine
           should_build: << pipeline.parameters.engine >>
           requires:
-            - model-build-lint-test
-
+            - designer-build-lint-test
```

Comment:
Why does engine now need designer-build-lint-test to have passed? It seems like engine should not need to wait this long (it only depends on model)?

Reply:
Because it was failing intermittently. The build is being unreliable, when running these in parallel.  This is an attempt to make it deterministic.

Reply:
Ok, thanks